cli/lib/kontena/cli/nodes/labels/add_command.rb

```diff
@@ -2,17 +2,17 @@ module Kontena::Cli::Nodes::Labels
   class AddCommand < Kontena::Command
     include Kontena::Cli::Common
 
-    parameter "NODE_ID", "Node id"
+    parameter "NODE", "Node"
```

Comment:
Should cli show node_id as a name if name is nil? Otherwise this might be a bit confusing to use...